apiserver/facades/client/cloud/cloud.go

```diff
@@ -28,112 +28,6 @@ import (
 
 var logger = loggo.GetLogger("juju.apiserver.cloud")
 
-// CloudV7 defines the methods on the cloud API facade, version 7.
-type CloudV7 interface {
```

Comment:
Personally I actually like these interface definitions. It shows clearly the expectations of the interface. However I know that most places don't follow this pattern. Do you not find them useful?

Reply:
No, I don't. They are bothersome to maintain, you need to know they are there for them to be of any use, otherwise you just end up adding methods to the facade and the interfaces become stale. And we only use them in this one place; we don't use them outside of this package to define a param type etc. So IMO they don't carry their weight. But I'll leave them in and we can revisit later.